sys-block/zram-init/zram-init-8.1-r2.ebuild

```diff
@@ -1,15 +1,14 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
```

Comment:
Please sync the live ebuild too.

Reply:
It seems to me that the live ebuild is already synced with `=zram-init-11.1` and both have a correct message.

Reply:
Oh, sorry, I missed you're changing the old version.. why bother? Just stable something newer (file a bug for that) and cleanup?

Reply:
So, I close this pull request and file a bug.